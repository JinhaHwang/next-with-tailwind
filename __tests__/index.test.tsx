import { render, screen } from '@testing-library/react'
import Home from '@/pages/index'

describe('Home', () => {
  it('renders a hello', () => {
    render(<Home />)

    const helloElement = screen.getByText(/hello/i)

    expect(helloElement).toBeInTheDocument()
  })

  it('renders homepage unchanged', () => {
    const { container } = render(<Home />)
    expect(container).toMatchSnapshot()
  })
})
