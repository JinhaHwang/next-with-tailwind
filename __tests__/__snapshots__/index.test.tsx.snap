// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Home renders homepage unchanged 1`] = `
<div>
  <div
    class="flex flex-col bg-red-500 px-8 text-3xl font-bold underline"
  >
    hello
  </div>
</div>
`;
